tests/tests_unit/test_data_classes/test_data_models/test_views.py

```diff
@@ -52,19 +52,62 @@ def test_load_dumped_mapped_property_for_apply(self) -> None:
             "source": {"space": "mySpace", "external_id": "myExternalId", "version": "myVersion", "type": "view"},
         }
 
-    def test_load_dump_connection_property(self) -> None:
+    def test_load_dump_single_edge_connection_property(self) -> None:
```

Comment:
load/dump roundtrip tests are actually generated for subclasses of CognitrObject, so these are redundant.

Reply:
removed tests using the default behaviour covered by the generated tests.